From: Wolfgang Fischer <wdev@posteo.org>
Date: Sun, 28  Jul 2024 18:05 +0000
Subject: [PATCH] fix install with DESTDIR

step-cli 0.27.0 changed the default install path to /ust/local.
Fix the Makefile so that the DESTDIR is used to create the install directory

---

diff --git a/Makefile b/Makefile
index b2fdf70a..623a0b95 100644
--- a/Makefile
+++ b/Makefile
@@ -128,7 +128,7 @@ govulncheck:
 INSTALL_PREFIX?=/usr/local/

 install: $(PREFIX)bin/$(BINNAME)
-	$Q mkdir -p $(INSTALL_PREFIX)bin/
+	$Q mkdir -p $(DESTDIR)$(INSTALL_PREFIX)bin
 	$Q install $(PREFIX)bin/$(BINNAME) $(DESTDIR)$(INSTALL_PREFIX)bin/$(BINNAME)

 uninstall:

